requirements/test-ci-default.txt

```diff
@@ -14,6 +14,7 @@
 -r extras/gevent.txt
 -r extras/elasticsearch.txt
 -r extras/couchdb.txt
+-r extras/couchbase.txt
```

Comment:
I think we also need to start setting an exact value of the dependencies in the [requirements](https://github.com/celery/celery/blob/master/requirements/extras/couchbase.txt) file and also specify the couchbase_ffi for Pypy?

Reply:
You are correct.